ebi_eva_common_pyutils/spring_properties.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2022 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from ebi_eva_common_pyutils.config_utils import get_primary_mongo_creds_for_profile, get_accession_pg_creds_for_profile,\
+    get_count_service_creds_for_profile
+
+
+class SpringPropertiesGenerator:
+    """
```

Comment:
#optional #standup I am wondering if we should take this opportunity to try [layered application properties](https://docs.spring.io/spring-boot/docs/2.1.0.RELEASE/reference/html/boot-features-external-config.html) to run pipelines i.e., use a 
1) "base properties file" which has the connection details/passwords/credentials for a given profile and can be locked down with proper permissions so that we don't proliferate these details across many places (see accession-production-datasources.properties under eva-accession directory in the cluster for references)
2) pipeline-specific properties file which this module can generate